features/conditions/membership_expiration_alerts.feature

```diff
@@ -0,0 +1,106 @@
+      | member02_exp_jan_4@mutts.se      | 5562252998     | rehab      | accepted |
+      | member03_exp_jan_15@mutts.se     | 5562252998     | rehab      | accepted |
+      | member04_exp_jan_15@mutts.se     | 5562252998     | rehab      | accepted |
+      | member05_exp_jan_31@mutts.se     | 5562252998     | rehab      | accepted |
+      | member06_exp_jan_31@bowwowwow.se | 2120000142     | rehab      | accepted |
+      | member07_exp_jan_31@bowwowwow.se | 2120000142     | rehab      | accepted |
+      | member08_exp_mar_2@bowwowwow.se  | 2120000142     | rehab      | accepted |
+      | member09_exp_mar_2@bowwowwow.se  | 2120000142     | rehab      | accepted |
+      | member10_exp_mar_2@bowwowwow.se  | 2120000142     | rehab      | accepted |
+      | member11_exp_mar_2@bowwowwow.se  | 2120000142     | rehab      | accepted |
+      | member12_exp_mar_2@bowwowwow.se  | 2120000142     | rehab      | accepted |
+
+
+    And the following payments exist
+      | user_email                       | start_date | expire_date | payment_type | status | hips_id | company_number |
+      | member01_exp_jan_3@mutts.se      | 2018-1-2   | 2019-1-3    | member_fee   | betald | none    |                |
+      | member02_exp_jan_4@mutts.se      | 2018-1-3   | 2019-1-4    | member_fee   | betald | none    |                |
+      | member03_exp_jan_15@mutts.se     | 2018-1-16  | 2019-1-15   | member_fee   | betald | none    |                |
+      | member04_exp_jan_15@mutts.se     | 2018-1-16  | 2019-1-15   | member_fee   | betald | none    |                |
+      | member05_exp_jan_31@mutts.se     | 2018-2-1   | 2019-1-31   | member_fee   | betald | none    |                |
+      | member06_exp_jan_31@bowwowwow.se | 2018-2-1   | 2019-1-31   | member_fee   | betald | none    |                |
+      | member07_exp_jan_31@bowwowwow.se | 2018-2-1   | 2019-1-31   | member_fee   | betald | none    |                |
+      | member08_exp_mar_2@bowwowwow.se  | 2018-3-3   | 2019-3-2    | member_fee   | betald | none    |                |
+      | member09_exp_mar_2@bowwowwow.se  | 2018-3-3   | 2019-3-2    | member_fee   | betald | none    |                |
+      | member10_exp_mar_2@bowwowwow.se  | 2018-3-3   | 2019-3-2    | member_fee   | betald | none    |                |
+      | member11_exp_mar_2@bowwowwow.se  | 2018-3-3   | 2019-3-2    | member_fee   | betald | none    |                |
+      | member12_exp_mar_2@bowwowwow.se  | 2018-3-3   | 2019-3-2    | member_fee   | betald | none    |                |
+
+
+
+  Scenario: On Jan 1, Membership Expire Alerts condition is processed
+    Given the date is set to "2019-01-01"
+    And there is a condition with class_name "MembershipExpireAlert" and timing "before"
+    And the condition has days set to [60, 30, 14, 2]
+    And the process_condition task sends "condition_response" to the "MembershipExpireAlert" class
```

Comment:
Nice to have these steps - very readable too.

Reply:
I'm sure we'll revise them as we go.  But it's a start and lets us begin to build up some good testing cases/data.